pygt3x/reader.py

```diff
@@ -291,16 +305,36 @@ def _get_data(self, num_rows=None):
         else:
             acceleration, temperature = self._get_data_default(num_rows=num_rows)
 
+        # Check for and remove identical samples
```

Comment:
@kitbeyer please check this

Reply:
Looks good to me - I checked with our data and the same outcome is accomplished, but I think you are correct, this is a more appropriate point to perform the check and removal - I added a couple suggested changes to the wording of the warning statements to indicate it was removing entire duplicate records rather than individual samples.